bin/osg-notify

```diff
@@ -185,11 +186,13 @@ def main():
 ===================================================
 """ + contents
 
-    msg = generateFullMessage(subject=args.subject, to=recipients, from_name=args.from_name, message=contents,
-                              sign=args.sign, keyid=args.keyid)
+    msgs = []
+    for recipient in recipients:
+        msgs.append(generateFullMessage(subject=args.subject, to=recipient, from_name=args.from_name, message=contents,
+                              sign=args.sign, keyid=args.keyid))
 
     if args.dryrun:
-        print(msg)
+        print('\n'.join(msg for msg in msgs))
```

Comment:
This is gonna print out a huge amount of output. I suggest doing the following instead:
```
if args.dryrun:
    if not recipients:
        raise Exception("Message has no recipients!")
    print("Recipients: " + ", ".join(recipients))
    print("Example message:")
    print(msg[0])
```

and making the same change below (after the "Not sending email...")

Reply:
ok, thanks, I will modify it